packages/squiggle-lang/src/PointSet/Discrete.ts

```diff
@@ -276,23 +279,6 @@ export const combinePointwise = <E>(
   );
 };
 
-export const reduce = <E>(
```

Comment:
This function was used only in `combineAlgebraically`, and it was used incorrectly (it computed `empty - t1 - t2` instead of `t1 - t2` on mixed pointsets).